app/client/build_runner.py

```diff
@@ -146,8 +146,8 @@ def _download_and_extract_results(self, timeout=None):
         """
         timeout_time = time.time() + timeout if timeout else sys.maxsize
 
-        download_artifacts_url = self._master_api.url('build', self._build_id, 'result')
-        download_filepath = 'build_results/artifacts.tar.gz'
+        download_artifacts_url = self._master_api.url('build', self._build_id, 'artifacts.zip')
```

Comment:
There are a number of places where `artifacts.zip` is referenced in the code. In addition, there is a new static define BuildArtifact.ARTIFACT_ZIPFILE_NAME defined as `results.zip`. The usage of the term "artifact" is getting pretty muddy.